pgxn/neon/pagestore_client.h

```diff
@@ -20,12 +20,25 @@
 #include RELFILEINFO_HDR
 #include "storage/block.h"
 #include "storage/smgr.h"
+#include "storage/buf_internals.h"
 #include "lib/stringinfo.h"
 #include "libpq/pqformat.h"
 #include "utils/memutils.h"
 
 #include "pg_config.h"
 
+#define MAX_SHARDS 128
+#define STRIPE_SIZE (256 * 1024 / 8) /* TODO: should in betaken from control plane? */
```

Comment:
Yes, I think we need to take the stripe size as a configuration input -- we'll probably learn more over time about what the best stripe size to use is, and adjust it.

Reply:
Done